keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -427,6 +439,15 @@ parametersRef:
   name: acme-gpu-init
 ```
 
+#### Integration with cluster autoscaling
+
+As a cloud provider, I want to support GPUs as part of a hosted Kubernetes
```

Comment:
I don't like the implications of this user story - I think it implies the DRA will only ever work in hosted environments such as GKE and will not be available in self-hosted clusters running it cloud (you even explicitly mention hosted Kubernetes environment). And it also implies every provider will likely need to their own version of integration, resulting in inconsistent experience for anyone using multi-cloud.

Can we explore what would happen if we changed this to "as a hardware vendor, I want to make my hardware available for applications that run in autoscaled cloud environments". This obviously makes it much harder to add a plugin in the first place, but it's also the most realistic way to guarantee support across cloud environments.